zoo-css-challenge/style.css

```diff
@@ -1,5 +1,14 @@
 * {
   font-family: sans-serif;
+
+  /* 4) Giraffes: Set font size in the universal selector to 
+        ensure the default size is 16px regardless of browser setting */
+  font-size: 16px;
```

Comment:
```suggestion
```
We should never override the browser font size setting, as this means if the user changes their browser setting font size to `24px` then this zoo website doesn't increase in size. Also, it's good practice to use relative units like `%` or `rem` for `font-size` instead of `px` so elements change proportionally when the user changes font size.

Reply:
I did not consider the user changing the size in the browser(potentially for accessibility reasons), thanks for pointing this out :)